frontend/public/components/search-filter-dropdown.tsx

```diff
@@ -16,82 +20,84 @@ export enum searchFilterValues {
   Name = 'Name',
 }
 
-export const SearchFilterDropdown: React.SFC<SearchFilterDropdownProps> = (props) => {
+export const SearchFilterDropdown: React.FC<SearchFilterDropdownProps> = ({
+  labelFilterInput,
+  nameFilterInput,
+  onChange,
+}) => {
   const [isOpen, setOpen] = React.useState(false);
   const [selected, setSelected] = React.useState(searchFilterValues.Label);
-
-  const { onChange, nameFilterInput, labelFilterInput } = props;
-
   const { t } = useTranslation();
 
-  const onToggle = (_event, open: boolean) => setOpen(open);
-  const onSelect = (event: React.SyntheticEvent) => {
-    setSelected((event.target as HTMLInputElement).name as searchFilterValues);
-    setOpen(!isOpen);
+  const onToggle = () => setOpen(!isOpen);
+  const onSelect = (event: React.SyntheticEvent, value: string) => {
+    setSelected(value as searchFilterValues);
+    setOpen(false);
   };
-  const dropdownItems = [
-    <DropdownItemDeprecated
-      key="label-action"
-      data-test="label-filter"
-      name={searchFilterValues.Label}
-      component="button"
-    >
-      {t(searchFilterValues.Label)}
-    </DropdownItemDeprecated>,
-    <DropdownItemDeprecated
-      key="name-action"
-      data-test="name-filter"
-      name={searchFilterValues.Name}
-      component="button"
-    >
-      {t(searchFilterValues.Name)}
-    </DropdownItemDeprecated>,
-  ];
   const handleKeyDown = (e: React.KeyboardEvent<HTMLInputElement>) => {
     if (e.key === 'Enter') {
       const { value } = e.target as HTMLInputElement;
       onChange(selected, value, true);
     }
   };
-
   const handleInputValue = (_event, value: string) => {
     onChange(selected, value, false);
   };
 
+  const selectItems = [
+    <SelectOption key="label-action" data-test="label-filter" value={searchFilterValues.Label}>
+      {t(searchFilterValues.Label)}
+    </SelectOption>,
+    <SelectOption key="name-action" data-test="name-filter" value={searchFilterValues.Name}>
+      {t(searchFilterValues.Name)}
+    </SelectOption>,
+  ];
+
   return (
-    <div className="pf-v5-c-input-group">
-      <DropdownDeprecated
-        onSelect={onSelect}
-        toggle={
-          <DropdownToggleDeprecated
-            id="toggle-id"
-            onToggle={onToggle}
-            toggleIndicator={CaretDownIcon}
+    <>
```

Comment:
one more nit: i don't think these react.fragments are necessary anymore since the entire component is wrapped by an `InputGroup`